psi4/src/psi4/libmints/orthog.cc

```diff
@@ -29,205 +29,165 @@
 #include <cstdlib>
 #include <cmath>
 #include <iterator>
+#include <algorithm>
 
 #include "psi4/psi4-dec.h"
+#include "psi4/libpsi4util/PsiOutStream.h"
 
-#include "matrix.h"
-#include "vector.h"
-#include "orthog.h"
+#include "psi4/libmints/matrix.h"
```

Comment:
These includes were actually fine.

Reply:
ok, my fault

Reply:
Should I revert?

Reply:
I don't think we have strict rules in place and both forms are fine. I've been cleaning up includes in the files I've opened lately and this jumped to my eyes.

Reply:
I guess this is resolved then?